go.sum

```diff
@@ -205,11 +244,17 @@ github.com/hashicorp/terraform v0.12.0 h1:It2vmod2dBMB4+r+aUW2Afx0HlftyUwzNsNH3I
 github.com/hashicorp/terraform v0.12.0/go.mod h1:Ke0ig9gGZ8rhV6OddAhBYt5nXmpvXsuNQQ8w9qYBZfU=
```

Comment:
References to `github.com/hashicorp/terraform` also need to be removed from `go.sum` - this can be done by running `go mod tidy` after removing lines manually from `go.mod`.